src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -171,9 +171,11 @@ private static int doTypeSwitch(Object target, int startIndex, Object[] labels)
 
     /**
      * Bootstrap method for linking an {@code invokedynamic} call site that
-     * implements a {@code switch} on a target of an enum type.  The static
-     * arguments are an array of case labels which must be non-null and of type
-     * {@code String} (representing enum constants) or {@code Class}.
+     * implements a {@code switch} on a target of an enum type. The static
+     * arguments are used to encode the case labels associated to the switch
```

Comment:
`String` and `Class` should appear in code blocks perhaps, or link tags? Also, I think this text could be improved by splitting the sentence by using a bullet list:

```
The static arguments are used to encode the case labels associated to the switch
construct, where each label can be encoded in two ways:
* as a String value, which represents the name of the enum constant associated with the label
* as a Class value, which represents the enum type associated with a type test pattern
```

Reply:
Thanks! I've (tried to) updated the javadoc as suggested.